tests/service/data/SyntaxTree/AbstractKeyword.fs

```diff
@@ -0,0 +1,2 @@
+type X =
+    abstract Y : int
```

Comment:
Would it be easy to add new lines to these files? It'll show if a new line is considered to be inside the range of a node: it  sometimes may be and sometimes may not (e.g. there was/is a difference in how signatures and implementation files are parsed IIRC).